src/components/FormBarChartRadioButton/index.js

```diff
@@ -44,28 +44,27 @@ const FormBarChartRadioButton = ({
         const style = { color: highlightColor, backgroundColor: COLOR_SCHEME_DEFAULT[value] }
         return (
           <div key={value} className="formBarChartRadioButton">
-            <input
-              type="radio"
-              className="formBarChartRadioButton__input"
-              style={style}
-              id={`formBarChartRadioButton_${input.name}_${value}`}
-              onChange={() => input.onChange(value)}
-              checked={input && input.value.toString() === value.toString()}
-              value={value}
-            />
-            <label
-              className={'formBarChartRadioButton__text outcomes outcomes--categorical'}
-              htmlFor={`formBarChartRadioButton_${input.name}_${value}`}
-            >
-              <div className="formBarChartRadioButton__outcome" style={{ width: `${probability}%` }}>
-                <div
-                  className="outcome__bar--inner pull-left"
-                  style={style}
-                />
-              </div>
-              {radioLabel}&nbsp;
-              <DecimalValue value={probability} decimals={2} />%
-            </label>
+            <div className="outcome">{radioLabel}&nbsp;</div>
+            <div className="formBarChartRadioButton__wrapper">
+              <input
+                type="radio"
+                className="formBarChartRadioButton__input"
+                style={style}
+                id={`formBarChartRadioButton_${input.name}_${value}`}
+                onChange={() => input.onChange(value)}
```

Comment:
Extract the arrow change as a constant. Otherwise every time the component will be asked if they need to be re-rendered it will answer yes.